contributing-guides/translation-templates/see-also-mentions.md

```diff
@@ -0,0 +1,335 @@
+# Mentioning related pages
+
+Oftentimes when you have related commands, be it tools that you use in tandem or commands that are similar
+to the one in the current page, it is nice to leave a hit of it existing in the header.
+An example of this is `top` where it's good to have mentions of alternatives like `atop`, `glances`,
+`btop` and `btm`.
+In order to notify the user that such pages exist, we put a little notice in the base command's description.
+This file contains the translation templates of this notice.
+
+[en](#en) •
+[ar](#ar) •
+[bn](#bn) •
+[bs](#bs) •
+[ca](#ca) •
+[cs](#cs) •
+[da](#da) •
+[de](#de) •
+[es](#es) •
+[fa](#fa) •
+[fi](#fi) •
+[fr](#fr) •
+[hi](#hi) •
+[id](#id) •
+[it](#it) •
+[ja](#ja) •
+[ko](#ko) •
+[lo](#lo) •
+[ml](#ml) •
+[ne](#ne) •
+[nl](#nl) •
+[no](#no) •
+[pl](#pl) •
+[pt_BR](#pt_br) •
+[pt_PT](#pt_pt) •
+[ro](#ro) •
+[ru](#ru) •
+[sr](#sr) •
+[sv](#sv) •
+[ta](#ta) •
+[th](#th) •
+[tr](#tr) •
+[uk](#uk) •
+[uz](#uz) •
+[zh](#zh) •
+[zh_TW](#zh_tw)
+
+---
+
+### en
+
+```markdown
+See also: `example`.
+```
+
+---
+
+### ar
+
+```markdown
+> انظر أيضًا: `example`.
+```
+
+---
+
+### bn
+
+```markdown
+> আরও দেখুন: `example`।
+```
+
+---
+
+### bs
+
+```markdown
+> Pogledajte isto: `example`.
+```
+
+---
+
+### ca
+
+```markdown
+> Vegeu també: `example`.
+```
+
+---
+
+### cs
+
+```markdown
+> Viz také: `example`.
+```
+
+---
+
+### da
+
+```markdown
+
+```
+
+---
+
+### de
+
+```markdown
+> Siehe auch: `example`.
+```
+
+---
+
+### es
+
+```markdown
+> Vea también: `example`.
+```
+
+---
+
+### fa
+
+```markdown
+> همچنین : `example`.
+```
+
+---
+
+### fi
+
+```markdown
+Katso myös: `example`.
+```
+
+---
+
+### fr
+
+```markdown
+> Voir aussi : `example`.
+```
+
+---
+
+### hi
+
+```markdown
+> यह भी देखें: `example`।
+```
+
+---
+
+### id
+
+```markdown
+> Guarda anche: `example`.
+> Lihat juga: `example`.
```

Comment:
.

Reply:
I guess we need to choose one of the two.

Reply:
Guarda ache doesn't even translate in Google translate so I guess we have a winner.
```suggestion
> Lihat juga: `example`.
```